core/src/HIP/Kokkos_HIP_BlockSize_Deduction.hpp

```diff
@@ -55,6 +55,23 @@
 namespace Kokkos {
 namespace Experimental {
 namespace Impl {
+
+template <typename DriverType, bool, int MaxThreadsPerBlock, int MinBlocksPerSM>
+void iOccupancy(int *numBlocks, int blockSize, int sharedmem) {
```

Comment:
I don't think we use this format anywhere else. Maybe,
```suggestion
void hipOccupancy(int *numBlocks, int blockSize, int sharedmem) {
```
?